projects/ngx-translate-router/src/lib/localize-router.parser.ts

```diff
@@ -342,9 +346,13 @@ export abstract class LocalizeParser {
     if (!this._translationObject) {
```

Comment:
You will have your escape prefix in URL if you don't put this in `else` statement and you don't receive the json response.

Reply:
You are right! Fixing it right now ;)